### DISCUSSION TITLE
Getting Paper References in Continual Learning Strategies Documentation

### DISCUSSION THREAD

**Discussion by KevinG1002** [ContinualAI/avalanche#1009] [Feature Request]
Hello, 

First, thank you for your good work. Avalanche is great to work with. 

I was wondering: would it be possible to add the links of the reference paper(s) used to implement the Continual Learning training strategies found in this library to the documentation? I was thinking this could be done in a similar format as in the FLOWER library whereby in each strategy class, within, the __init__ function, a link to the reference seminal paper is provided. 

Thanks again for your work,

Kevin

**Comment by vlomonaco** [comment]
Hi @KevinG1002 thanks for the suggestion! Indeed it would be very useful. I think we do it already in the [reproducible CL](https://github.com/ContinualAI/reproducible-continual-learning) repo, right @AndreaCossu? 

This could be a nice "good-first" issue I guess :)

**Reply by AndreaCossu** [reply]
right :smile:
